metrics.md

```diff
@@ -0,0 +1,47 @@
+ 
+#### Gluster prometheus end point schema: 
+
+#### Volume Metrics : 
+
+|Gluster Promethus schema| example| codes |
+|--------|-----------|---|
+|gluster_peer_status |	gluster_peer_status{nodeid="xxxxx"} 1 |	1 - connected, 0 - disconnected	
+|gluster_volume_status|	gluster_volume_status{volume="xxx"} 1	| 1 - started, 0 - stopped, 2 - starting, 3 - stopping	
+|gluster_volume_fop_max_latency	| gluster_volume_max_latency{volume=""xxx"", brick_path=""xxxx"",fop=""xxx""}  800970.00| 	
+|gluster_volume_fop_min_latency	|gluster_volume_fop_min_latency{volume="xxx", brick_path="xxx", fop="xxx"} 21.00	|
+|gluster_volume_fop_avg_latency|	gluster_volume_fop_avg_latency{volume="xxx", brick_path="xxx", fop="xxx"} 1132.00|	
+|gluster_volume_fop_calls |gluster_volume_fop_calls{volume="xxx". brick_path="xxx", fop="xxx"} 13	|
+|gluster_volume_fop_percentage_latency| gluster_volume_fop_percentage_latency(volume="xxx", brick_path="xxx", fop="xxx"} 13	|
```

Comment:
peerid and brick host are required whenever a metric exported with brick path. Reason: brick_path can be same across the peers

Reply:
ack.